test/index/down.test.js

```diff
@@ -77,27 +77,6 @@ describe('down', function () {
 
     describe('when no option is specified', function () {
       beforeEach(function () {
-        return this.umzug.down().bind(this).then(function (migrations) {
-          this.migrations = migrations;
-        });
-      });
-
-      it('returns 1 item', function () {
```

Comment:
I have considered `down()` and `down({})` are the same test cases. In addition I'm refactoring this completely.